nmostesting/suites/IS0401Test.py

```diff
@@ -31,6 +31,7 @@
 from ..IS04Utils import IS04Utils
 from ..TestHelper import get_default_ip, is_ip_address, load_resolved_schema
 from ..mocks.Node import NODE
+from ..mocks.Auth import PRIMARY_AUTH

```

Comment:
To be consistent with use of the mock Registry and DNS server, the PRIMARY_AUTH should be injected into the test suite constructor (see line 570 onwards in `NMOSTesting.py`) rather than included as global.

Reply:
Right... and still not yet understanding why both `AUTHS` could not be injected this way?

https://github.com/AMWA-TV/nmos-testing/blob/5b3f1eca4db1f37d8828e37eae8e93f108529946/nmostesting/NMOSTesting.py#L596

Reply:
Done as @jonathan-r-thorpe suggestion

Reply:
@lo-simon Unfortunately you need to merge or rebase on master, since the mechanism you have now used has changed...